<launch>
	<arg
		name="device_name"
		default="kern_Door"/>
	<arg
		name="connection_mode"
		default="serial"/>
	<arg
		name="address"
		default="None"/>
	<arg
		name="port"
		default="/dev/ttyACM0"/>
	<arg
		name="simulation"
		default = 'False'/>
	<node
		name="Kern_door_driver"
		pkg="roslabware_drivers"
		type="kern_door"
		args="
			-d $(arg device_name)
			-c $(arg connection_mode)
			-add $(arg address)
			-p $(arg port)
			-l $(arg simulation)"
		output="screen"/>
</launch>
